enterprise/api/v1/serializers.py

```diff
@@ -129,10 +166,23 @@ def save(self):  # pylint: disable=arguments-differ
         )
 
 
+class EnterpriseCatalogSerializer(serializers.ModelSerializer):
```

Comment:
Argh, not sure how this slipped through the cracks while I was renaming; thanks.